HeterogeneousCore/CUDAUtilities/interface/host_noncached_unique_ptr.h

```diff
@@ -13,53 +13,60 @@ namespace cudautils {
         // Additional layer of types to distinguish from host::unique_ptr
         class HostDeleter {
         public:
-          void operator()(void *ptr) {
-            cuda::throw_if_error(cudaFreeHost(ptr));
-          }
+          void operator()(void *ptr) { cuda::throw_if_error(cudaFreeHost(ptr)); }
         };
-      }
+      }  // namespace impl
 
       template <typename T>
       using unique_ptr = std::unique_ptr<T, impl::HostDeleter>;
 
       namespace impl {
         template <typename T>
-        struct make_host_unique_selector { using non_array = cudautils::host::noncached::unique_ptr<T>; };
+        struct make_host_unique_selector {
+          using non_array = cudautils::host::noncached::unique_ptr<T>;
+        };
         template <typename T>
-        struct make_host_unique_selector<T[]> { using unbounded_array = cudautils::host::noncached::unique_ptr<T[]>; };
+        struct make_host_unique_selector<T[]> {
+          using unbounded_array = cudautils::host::noncached::unique_ptr<T[]>;
+        };
         template <typename T, size_t N>
-        struct make_host_unique_selector<T[N]> { struct bounded_array {}; };
-      }
-    }
-  }
+        struct make_host_unique_selector<T[N]> {
+          struct bounded_array {};
+        };
+      }  // namespace impl
+    }    // namespace noncached
```

Comment:
why do these get aligned ?

Reply:
IIRC we have a rule for aligning comments on consecutive lines. I would also prefer it to not apply for end-of-namespace comments.